src/main/kotlin/org/rust/ide/annotator/fixes/AddStructFieldsFix.kt

```diff
@@ -51,12 +54,21 @@ class AddStructFieldsFix(
         val firstAdded = defaultValueBuilder.fillStruct(
             body,
             decl.fields,
-            fieldsToAdd
+            fieldsToAdd,
+            this.getVisibleBindings(startElement)
         )
 
         if (editor != null && firstAdded != null) {
             val expr = firstAdded.expr
             if (expr != null) editor.caretModel.moveToOffset(expr.textOffset)
         }
     }
+
+    private fun getVisibleBindings(place: RsElement) : HashSet<RsPatBinding> {
+        val bindings = HashSet<RsPatBinding>()
```

Comment:
Actually I just need to iterate though this, so I might as well convert it to a `List`. I wanted to ask whether I should extract it to a separate function (it's used both here and in the initialize with default value fix). If it's extracted into some utility space (I don't know where), then it would make more sense to use the `Map` as you say :)

Reply:
@ortem I remember something similar in a one of your quickfies. Could you, please, point that place?